### PR TITLE
Replaced go-dep with go modules

### PR DESCRIPTION
**Pull request by basheuft**

### CODE REVIEW

File: check.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/bin/bash -e
-dep ensure
+#dep ensure
 #gometalinter --skip test --skip defaults --cyclo-over=15 --deadline=90s --vendor ./...
-golangci-lint run
+go mod tidy
+#golangci-lint run
```
Comment (Ulexus) [inline check.sh]: Clobbered linter

**Review by Ulexus — COMMENTED** [review]
Thanks; I've been meaning to get around to this, but since I keep everything in my GOPATH, I never get around to it.  Just one thing:  you clobbered the linter... was that intentional?

File: check.sh (hunk 1 above)
Comment (basheuft) [reply check.sh]: Ahh :(
I'll fix that. Didn't notice right away.

**Comment by Ulexus** [comment]
For the same reason that I haven't converted over yet, Travis is failing.  Can you add the `GO111MODULE=on` environment variable setting to the script?

**Comment by Ulexus** [comment]
Superseded by #4

**Pull request closed without merging**